app/src/main/java/org/fossasia/phimpme/trashbin/TrashBinActivity.java

```diff
@@ -62,6 +76,20 @@
         setUpUi();
     }
 
+    private void trashEmptyViewSetup(){
+        if(ThemeHelper.getBaseTheme(getApplicationContext()) == ThemeHelper.DARK_THEME ||
+                ThemeHelper.getBaseTheme(getApplicationContext()) == ThemeHelper.AMOLED_THEME){
+            emptyIcon.setImageResource(R.drawable.ic_delete_sweep_white_24dp);
+            trashText.setTextColor(getTextColor());
+            trashMssge.setTextColor(getTextColor());
+
+        } else if (ThemeHelper.getBaseTheme(getApplicationContext()) == ThemeHelper.LIGHT_THEME){
```

Comment:
We could change this to a single else. As if it's not the dark theme, let's assume it to be the light theme.